terraform/application/config/production.tfvars.json

```diff
@@ -18,5 +18,6 @@
   },
   "enable_monitoring": true,
   "statuscake_contact_groups": [195955, 282453],
-  "external_url": "https://claim-additional-payments-for-teaching-production-web.teacherservices.cloud/healthcheck"
+  "external_url": "https://claim-additional-payments-for-teaching-production-web.teacherservices.cloud/healthcheck",
+  "enable_logit": true
```

Comment:
Can we enable first on review and test? So we can validate before shipping prod logs